app/models/concerns/gias_helpers.rb

```diff
@@ -24,12 +24,14 @@ def name_and_urn
     "#{name} (#{urn})"
   end
 
-private
-
   def open?
     open_status_code?(school_status_code)
   end
 
+  def closed?
+    !open_status_code?(school_status_code)
+  end
```

Comment:
```
  def closed?
    !open
  end
```
?

Reply:
Yes, then I thought that's just an extra call .... :)